contrib/python/pypodman/pypodman/lib/parser_actions.py

```diff
@@ -63,6 +64,53 @@ def __call__(self, parser, namespace, values, option_string=None):
             setattr(namespace, self.dest, val)
 
 
+class ChangeAction(argparse.Action):
+    """Convert and validate change argument."""
+
+    def __init__(self,
+                 option_strings,
+                 dest,
+                 nargs=None,
+                 const=None,
+                 default=None,
+                 type=None,
+                 choices=None,
+                 required=False,
+                 help=None,
+                 metavar='OPT=VALUE'):
+        """Create ChangeAction object."""
+        help = (help or '') + ('Apply change(s) to the new image.'
+                               ' May be given multiple times.')
+
+        super().__init__(
```

Comment:
(Note: I don't know the code, nor remember if this is a python3 thing or not)

Doesn't this need to be ``super(ChangeAction, self).__init__(`` ?

Reply:
@cevich Python3 finally made super calls simple as all classes are now "new" classes.

Reply:
gotcha, thx.